### PR TITLE
u3d/install: add support for installing .msi packages on Windows

### PR DESCRIPTION
**Pull request by niezbop**
This adds support for downloading and installing .msi packages for Windows.

### CODE REVIEW

File: lib/u3d/installer.rb [hunk 1]
```diff
@@ -311,7 +311,11 @@ def install_exe(file_path, installation_path: nil, info: {})
         command = nil
         if info['cmd']
           command = info['cmd']
-          command.sub!(/{FILENAME}/, file_path)
+          if /msiexec/ =~ command
+            command.sub!(/{FILENAME}/, '"' + file_path.gsub(%r{\/(\d)}, '/\\\\\1').tr('/', '\\') + '"')
```
Comment (lacostej) [inline lib/u3d/installer.rb]: turn this into a private method?

windows_path(file_path) or something?

Comment (niezbop) [reply lib/u3d/installer.rb]: Maybe put in utils?
This could be reusable

Comment (lacostej) [reply lib/u3d/installer.rb]: as you wish. At least extract it from here to that we can understand what happens without decrypting the code :)

**Comment by lacostej** [comment]
I just realized that in the same file we already do some windows sanitization. In a different way:

```
      final_path = installation_path.tr('/', '\\')
```

**Comment by niezbop** [comment]
Oh right. Well this is not the good way then, let me change that. The issue we had was when sub was called with `\[digit]` which was causing issues with the download path. I'll use the new correct one then.

**Pull request merged**